datakyt/db/insert_data_in_project_table.py

```diff
@@ -0,0 +1,27 @@
+import psycopg2
+import csv
+import os
+
+
+def insert_csv_file_to_database(path, conn):
+    cur = conn.cursor()
+    with open(os.path.join(path), 'r') as f:
```

Comment:
Why do you need `os.path.join` here?

Reply:
If we use os.path.join, it helps avoid some mistakes, so I believe we could use this. When I wrote the path to the file from the desktop, os.path.join helped solve this problem. Since we have the csv file in the same folder as the function, we don't really need it. But if the file is located elsewhere, then os.path.join helps us not to get errors

Reply:
@OleksandrZhytnyk what problem? 
`os.path.join` is useless with 1 arg.